examples/basics/batches.ipynb

```diff
@@ -72,22 +71,29 @@
     {
       "metadata": {},
       "source": [
-        "# API Key and Client\n",
+        "## API key and client\n",
```

Comment:
Could you add "-q" to the !pip install "labelbox[data]" cell ? I forgot to add this.